stdlib/public/Concurrency/TaskGroup.swift

```diff
@@ -491,15 +487,17 @@ public struct TaskGroup<ChildTaskResult: Sendable> {
 
   /// Cancel all of the remaining tasks in the group.
   ///
-  /// After cancellation,
-  /// any new results from the tasks in this group
-  /// are silently discarded.
```

Comment:
This isn't true, cancelled tasks run as usual and can be collected.

FYI @amartini51 in case we used this assumption somewhere else in some docs?